python/graphstorm/model/node_glem.py

```diff
@@ -125,8 +143,23 @@ def set_gnn_encoder(self, gnn_encoder):
 
     @property
     def gnn_encoder(self):
```

Comment:
can you also add comment that these properties are only used for model inference and evaluation?